docs/maintenance.md

```diff
@@ -5,74 +5,82 @@ How to change the supported Kubernetes minor versions
 -------------------------------------------
 
 pvc-autoresizer depends on some Kubernetes repositories like `k8s.io/client-go` and should support 3 consecutive Kubernetes versions at a time.
+
 Issues and PRs related to the last upgrade task also help you understand how to upgrade the supported versions,
 so checking them together(e.g https://github.com/topolvm/pvc-autoresizer/pull/85) with this guide is recommended when you do this task.
 
-### Check release notes
-
-First of all, we should have a look at the release notes in the order below.
-
-1. TopoLVM
-    - Choose the [TopoLVM](https://github.com/topolvm/topolvm/releases) version that supported target Kubernetes version.
-2. Kubernetes
-    - Choose the next version and check the [release note](https://kubernetes.io/docs/setup/release/notes/). e.g. 1.17, 1.18, 1.19 -> 1.18, 1.19, 1.20
-    - Read the [release note](https://github.com/kubernetes-sigs/controller-runtime/releases), and check whether there are serious security fixes and whether the new minor version is compatible with older versions from the pvc-autoresizer's point of view. If there are breaking changes, we should decide how to manage these changes.
-    - Read the [kubebuilder go.mod](https://github.com/kubernetes-sigs/kubebuilder/blob/master/go.mod), and check the controller-tools version corresponding to controller-runtime.
-3. Depending tools
-    - They does not depend on other software, use latest versions.
-      - [helm](https://github.com/helm/helm/releases)
-      - [helm-docs](github.com/norwoodj/helm-docs/releases)
-      - [kube-prometheus](https://github.com/prometheus-operator/kube-prometheus/releases)
-4. Depending modules
-    - Read [kubernetes go.mod](https://github.com/kubernetes/kubernetes/blob/master/go.mod), and update the `prometheus/*` modules.
-5. Golang version
-    - Consider whether the golang should be updated.
+### Upgrade procedure
 
 We should write down in the github issue of this task what are the important changes and the required actions to manage incompatibilities if exist.
 The format is up to you.
 
 Basically, we should pay attention to breaking changes and security fixes first.
 
-### Update written versions
+#### TopoLVM
+
+Choose the [TopoLVM](https://github.com/topolvm/topolvm/releases) version that supported target Kubernetes version.
 
-We should also update the following files.
+To change the version, edit `e2e/Makefile`. If TopoLVM which support the target Kubernetes version has not released yet, you can specify the commit hash instead of the tag.
```

Comment:
```suggestion
To change the version, edit `e2e/Makefile`. If TopoLVM which supports the target Kubernetes version has not released yet, you can specify the commit hash instead of the tag.
```